src/api/api.tsx

```diff
@@ -5,68 +5,154 @@ import {
   NotFound,
   PermissionDenied,
 } from '../errors/errors';
-
-const LOCALSTORAGE_TOKEN_NAME = 'zaaktypenbeheer_token';
-localStorage.setItem(LOCALSTORAGE_TOKEN_NAME, import.meta.env.VITE_API_LOGIN_TOKEN);
-
-interface APIResponse {
-  detail: string;
-  [key: string]: any;
-}
+import { CSRFToken } from './headers';
 
 class Api {
-  api_token: string | null;
   api_url: string;
```

Comment:
Shouldn't this be camelCase? Also, I think maybe apiBaseUrl could be a better name :thinking: 

I also wonder if it is needed to have the api methods bundled inside the class `Api` instead of having them as just functions.

Reply:
I will change the names

Reply:
For code readability it is cleaner to see `api.gt(endpoint)` rather than `get(endpoint)`. But i will change this to separate functions instead of one `Api` class